docs/integrations/destinations/azureblobstorage.md

```diff
@@ -137,4 +138,5 @@ They will be like this in the output file:
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
 | 0.1.0 | 2021-08-30 | [\#5332](https://github.com/airbytehq/airbyte/pull/5332) | Initial release with JSONL and CSV output. |
+| 0.1.1 | 2021-12-29 | [\#5332](https://github.com/airbytehq/airbyte/pull/9190) | Added BufferedOutputStream wrapper to blob output stream to improve performance and fix issues with 50,000 block limit. Also disabled autoflush on PrintWriter. |
```

Comment:
We usually put the latest changes at the top of the table.

Reply:
No problem, I'll get that fixed